ipython/ResamplingAndMonteCarloMethods/resampling_tutorial_3.ipynb

```diff
@@ -146,7 +146,7 @@
    "source": [
     "The two standard errors estimates differ slightly because the bootstrap algorithm is inherently stochastic, but that is OK. The best we can hope for is an approximation, and these two approximations agree with one another quite well.\n",
     "\n",
-    "An even better way of quantifying the uncertainty, especially when the distribution of the statisic is non-normal, is to produce a *confidence interval* on the statistic. Suppose we perform the the experiment repeatedly and produce a \"95% confidence interval\" $l_i$ and $u_i$ from the data in each experiment $i$; this means that the true value of the statistic (the difference in the *population* means) will be between $l_i$ and $u_i$ in 95% of the replications $i$."
+    "An even better way of quantifying the uncertainty, especially when the distribution of the statisic is non-normal, is to produce a *confidence interval* on the statistic. Suppose we perform the experiment repeatedly and produce a \"95% confidence interval\" $(l_i, u_i)$ from the data in each experiment $i$; this means that we should expect the true value of the statistic (the difference in the *population* means) will be between $l_i$ and $u_i$ in 95% of the replications $i$."
```

Comment:
```suggestion
    "An even better way of quantifying the uncertainty, especially when the distribution of the statisic is non-normal, is to produce a *confidence interval* on the statistic. Suppose we perform the experiment repeatedly and produce a \"95% confidence interval\" $(l_i, u_i)$ from the data in each experiment $i$; this means that we should expect the true value of the statistic (the difference in the *population* means) to be between $l_i$ and $u_i$ in 95% of the replications $i$."
```

Reply:
That's a good point about people not knowing the tutorials by number. Maybe just 
"the previous tutorial on [Permutation Tests](link)."?